src/main/java/twilightforest/structures/StructureTFMajorFeatureStart.java

```diff
@@ -345,7 +345,7 @@ public boolean isLocked(int lockIndex) {
             //System.out.println("Checking locks for lockIndex " + lockIndex);
 
             for (int i = 0; i < this.lockBytes.length; i++) {
-                System.out.println("Lock " + i + " = " + this.lockBytes[i]);
+                //System.out.println("Lock " + i + " = " + this.lockBytes[i]);
```

Comment:
just remove the loop totally. No point in keeping an empty loop to spin, and the code looks like it's a debug code, so we might as well remove it. If it was needed it can be found in the git history anyways.

Reply:
Sounds good!